### ISSUE TITLE
About config

### ISSUE THREAD

**Issue by Yangr116** [zhang-tao-whu/e2ec_mmdet#1]
Hi, I would like to confirm that ```e2ec_fcos_center-normbbox-centeronreg-giou_r50_caffe_fpn_gn-head_dcn_feature64_p4init_p4evolve_dml_1x_coco.py``` and ```e2ec_fcos_center-normbbox-centeronreg-giou_r50_caffe_fpn_gn-head_dcn_feature64_p4init_p4evolve_dml_mstrain_4x_coco.py``` are the version of your paper, except the detector?

**Comment by zhang-tao-whu** [comment]
Yes. However, the generation process of the initial contours may have been slightly modified to be compatible with the multi-scale detection head of FCOS.
